windows/src/engine/mcompile/mc_crc32.cpp

```diff
@@ -1,5 +1,5 @@
 
-#include "stdafx.h"
+#include "pch.h"
```

Comment:
Because our global C++ classes are not yet in a shared library, we have to do dumb things like this where precompiled headers are inconsistent between projects. Yuck. Another possible groundwork project for 15.0?